src/app/app.component.ts

```diff
@@ -2,29 +2,42 @@
  * Angular 2 decorators and services
  */
 import { Component, ViewEncapsulation } from '@angular/core';
+import { cloneDeep } from 'lodash';
+import { ActivatedRoute } from '@angular/router';
+import { AuthenticationService, Broadcaster } from 'ngx-login-client';
+import { Notification, NotificationActionCallback } from 'ngx-fabric8-wit';
```

Comment:
Are you not able to use the Notification object from ngx-widgets? Is it because you want to provide your own callbacks? Just curious how we can improve the component?